src/main/java/com/iotiq/application/domain/CartItem.java

```diff
@@ -0,0 +1,31 @@
+package com.iotiq.application.domain;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.ManyToOne;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import org.springframework.data.jpa.domain.AbstractPersistable;
+
+import java.util.UUID;
+
+@Entity
+@Getter
+@Setter
+@NoArgsConstructor
+@AllArgsConstructor
+public class CartItem extends AbstractPersistable<UUID> {
+
+    @ManyToOne
+    @JoinColumn(name = "product_id")
```

Comment:
It's actually totally up to you to use a JoinColumn but in fact we don't need it here. ManyToOne, by default, uses a JoinColumn and uses the Id column for that, so using this has no effect.